src/main/java/net/devtech/arrp/json/recipe/JStonecuttingRecipe.java

```diff
@@ -0,0 +1,25 @@
+package net.devtech.arrp.json.recipe;
+
+public class JStonecuttingRecipe extends JRecipe {
+    private final JIngredient ingredient;
+    private final String result;
+    private final int count;
```

Comment:
is the count field optional?, if so make it an Integer so GSON will omit it

Reply:
Same goes for all the other primitives

Reply:
from the wiki:
```
Unlike the count field in shaped and shapeless crafting recipes, this count field here is required. 
```

Reply:
oh ok, that's fine then